### PR TITLE
tests: support parameters for match-log on journal-state tool

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The idea is to improve match-log to be able to remove all the loops in
the code using get-log instead

### CODE REVIEW

File: tests/lib/tools/journal-state [hunk 1]
```diff
@@ -40,22 +40,50 @@ check_log_started(){
 }
 
 match_log(){
-    expression=$1
-    shift
-    for _ in $(seq 10); do
+    local wait=1
+    local attempts=10
```
Comment (stolowski) [inline tests/lib/tools/journal-state:44]: IMHO the defaults should be wait=0 and attempts=1 (meaning we just match current log state without sleep/retry).

**Review by stolowski — APPROVED** [review]
LGTM, thank you

**Review by bboozzoo — APPROVED** [review]
LGTM

File: tests/lib/tools/journal-state (hunk 1 above)
Comment (sergiocazzolato) [reply tests/lib/tools/journal-state:44]: Nice, thanks for the review

**Review by mardy — APPROVED** [review]

**Pull request merged**